layers/+lang/scheme/packages.el

```diff
@@ -106,3 +114,39 @@
 
 (defun scheme/post-init-parinfer ()
   (add-hook 'scheme-mode-hook 'parinfer-mode))
```

Comment:
Can you update this line and L31?
`parinfer-mode` has been superseded by `parinfer-rust-mode`.

Reply:
Well I also found three other layers are using the deprecated `parinfer-mode`.

Reply:
Sure... done!

Reply:
But it is a post-init function. It does not install parinfer-rust-mode (and also parinfer-rust-mode does not already get installed by another layer here)